docs/pages/eas/metadata/config.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Create config files for EAS Metadata
```

Comment:
```suggestion
title: Create config files for EAS Metadata
```
This is a bit wordsy, maybe 'Configuring EAS Metadata' like we do with app.json

Reply:
```suggestion
title: Configuring EAS Metadata
```